cash_private.h

```diff
@@ -27,6 +27,16 @@
 #define CASHSERVER_TOF_CONF_FILE	"/vendor/etc/tof_focus_calibration.xml"
 #define CASHSERVER_RGBC_CONF_FILE	"/vendor/etc/cash_expcol_calibration.xml"
 
+#define CASHSERVER_DATASTORE_DIR	"/data/vendor/misc/cashsvr/"
```

Comment:
`/data/vendor/cashsvr` makes more sense now that I see the structure. `/data/misc/cashsvr` would have made sense under `/data/misc`, but nothing uses `/data/vendor/misc/*`
Either way, you need to add a `mkdir` in the cashsvr.rc file as well.